apidef/oas/security.go

```diff
@@ -515,10 +563,16 @@ func (c *IntrospectionCache) ExtractTo(cache *apidef.IntrospectionCache) {
 	cache.Timeout = c.Timeout
 }
 
+// ExternalOAuth holds configuration for an external OAuth provider.
 type ExternalOAuth struct {
-	Enabled     bool `bson:"enabled" json:"enabled"` // required
+	// Enabled activates external oauth functionality.
```

Comment:
Should add required status to description in this field and in fields below where required comment appears?